RELEASE.md

```diff
@@ -65,7 +106,9 @@ March 4th, 2015 | [Release Notes](https://predictionio.atlassian.net/jira/secure
 
 ###v0.8.6
 
-Feb 10th, 2015 | [Release Notes](https://predictionio.atlassian.net/jira/secure/ReleaseNote.jspa?projectId=10000&version=13300)
+Feb 10th, 2015
+
+[Release Notes](https://predictionio.atlassian.net/jira/secure/ReleaseNote.jspa?projectId=10000&version=13300)
 
 - New engine template - [Product Ranking](/templates/productranking/quickstart/) for personalized product listing
 - [CloudFormation deployment](/system/deploy-cloudformation/) available
```

Comment:
Slight reformat and added AML history that is the subject of this PR.

Ready for review